.github/workflows/test.yml

```diff
@@ -63,7 +63,9 @@ jobs:
 
             - name: Installing dependencies
               if: steps.carthage-cache.outputs.cache-hit != 'true'
-              run: bash carthage-build.sh --no-use-binaries --platform iOS --cache-builds
+              run: |
+                  carthage update --new-resolver --no-build
+                  bash carthage-build.sh update --no-use-binaries --platform ios --cache-builds
```

Comment:
Why both of these? And should the first `carthage update` use the script?

Reply:
Looks like it works with just the second line.